pom.xml

```diff
@@ -62,12 +62,19 @@
           <version>${plugins.job-dsl.version}</version>
           <optional>true</optional>
         </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-test-harness-htmlunit</artifactId>
+      <version>2.18-1</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
```

Comment:
Any reason to not use the [apache-httpcomponents-client-4-api](https://plugins.jenkins.io/apache-httpcomponents-client-4-api/) plugin here instead?